### ISSUE TITLE
Arrow key to cycle lightbox images also scrolls columns

### ISSUE THREAD

**Issue by otterlove** [tootsuite/mastodon#5766]
Steps to reproduce:

1. Add several columns and/or make the browser width smaller so a horizontal scroll bar appears.
2. Click on one of several images in a post to expand it.
3. Use the left and right arrow keys to cycle the images.

Result: the columns behind the images also shift while the key is pressed. Seen in Chrome.
Expected: Elements behind an overlay should be disabled while the overlay is active.

* * * *

- [X] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by averageunicorn** [comment]
This happens in Firefox as well.

**Comment by ilianaw** [comment]
I'd suggest that this is an accessibility issue, as it causes the "background" to shift under the content you're viewing unexpectedly.

**Comment by ykzts** [comment]
Resolved with #6241
